javatests/arcs/android/host/TestBindingDelegate.kt

```diff
@@ -0,0 +1,41 @@
+package arcs.android.host
+
+import android.content.Context
+import android.content.ServiceConnection
+import arcs.android.storage.ParcelableStoreOptions
+import arcs.sdk.android.storage.service.StorageService
+import arcs.sdk.android.storage.service.StorageServiceBindingDelegate
+import org.robolectric.Robolectric
+import org.robolectric.android.controller.ServiceController
+
+class TestBindingDelegate(val context: Context) :
+    StorageServiceBindingDelegate {
```

Comment:
Prefer this styling:

```suggestion
class TestBindingDelegate(
    val context: Context
) : StorageServiceBindingDelegate {
```